### ISSUE TITLE
Are the leaderboard entries using accuracy or top k accuracy?

### ISSUE THREAD

**Issue by Mirandatz** [fastai/imagenette#39]
I am asking because the entry:

256 | 20 | fastai2 train_imagenette.py 2020-01 + MaxBlurPool | 93.53% | 5, mean

Related to the following ipynb

https://github.com/ducha-aiki/imagewoofv2-fastv2-maxpoolblur/blob/master/fastai2-imagenette-train-maxblurpool.ipynb

Appears with accuracy "93.53%" in the leader board, but in the ipynb the accuracy never reaches 86%.
What does reach >90% is the top_k_accuracy.

**Comment by tmabraham** [comment]
Under the [Tips](https://github.com/fastai/imagenette#tips) section, it is mentioned:

> Because there are only 10 categories, the usual "top 5 accuracy" isn't so interesting. So you should generally report top 1 accuracy when using Imagenette

So all score are top-1 accuracy.

I think maybe that link isn't the best link. The logs for that experiment is apparently provided [here](https://github.com/ducha-aiki/imagewoofv2-fastv2-maxpoolblur/blob/master/log_woof_5_ep_192_blurpool.log) and the parsed results are [here](https://github.com/ducha-aiki/imagewoofv2-fastv2-maxpoolblur/blob/master/parse_logs_and_print_mean.ipynb).

**Comment by Mirandatz** [comment]
> Under the [Tips](https://github.com/fastai/imagenette#tips) section, it is mentioned:
> 
> > Because there are only 10 categories, the usual "top 5 accuracy" isn't so interesting. So you should generally report top 1 accuracy when using Imagenette
> 
> So all scores are top-1 accuracies.
> 
> I think maybe that link isn't the best link. The logs for that experiment is apparently provided [here](https://github.com/ducha-aiki/imagewoofv2-fastv2-maxpoolblur/blob/master/log_woof_5_ep_192_blurpool.log) and the parsed results are [here](https://github.com/ducha-aiki/imagewoofv2-fastv2-maxpoolblur/blob/master/parse_logs_and_print_mean.ipynb).

I see. It appears the specific log is this one:
https://github.com/ducha-aiki/imagewoofv2-fastv2-maxpoolblur/blob/master/logs/log_nette_20_ep_256_blurpool.log

Now I am trying to understand the training script [train.py](https://github.com/ducha-aiki/imagewoofv2-fastv2-maxpoolblur/blob/master/train.py) 

If I understand it correctly, the [xresnet50 architecture](https://github.com/ducha-aiki/imagewoofv2-fastv2-maxpoolblur/blob/af6eac8060ed4f11c52b9895d0ed71b928d765ef/train.py#L57) was used as a base, right?

[Since the xresnet50 default value for "pretrained" is false](https://docs.fast.ai/vision.models.xresnet.html#xresnet50),  this means that just the architecture was utilized, not the weights, right?

**Comment by Mirandatz** [comment]
@tmabraham Thank you for the quick response.
Precisely because of the _Tips sections_, and because the link in the table redirects to a sub-optimal notebook, that I asked.

Another (possibly silly) question.
I am not familiar with the fastai library; so I'm not sure how to interpret a log like [this](https://raw.githubusercontent.com/ducha-aiki/imagewoofv2-fastv2-maxpoolblur/master/logs/log_nette_20_ep_192_blurpool.log).

I can easily see that the values in column 1 is related to the train dataset and that the values in column 2 are related to the validation dataset.
What about the values in columns 3 (accuracy) and 4 (top_k_accuracy)?
Are they measured in the train dataset or the validation dataset?

**Comment by tmabraham** [comment]
@Mirandatz no worries! Glad to hear I addressed your confusion.

All metrics in fastai are computed on the validation set. So the accuracy and top_k_accuracy values are indeed for the validation set.

**Comment by tmabraham** [comment]
BTW @hamelsmu this issue can probably be closed, I addressed the questions here...
